test_extract_unique_sensors_01.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+from extract_unique_sensors_01 import extract_all_sensors
+from extract_unique_sensors_01 import splitting_extracted_sensors
+
+
+class TestExtractSensors(unittest.TestCase):
+    """Test class for function test_read_sensors."""
+
+    def test_extract_all_sensors(self):
+        sensor_name = "kat"
+        sensors_data = [
+            "kat.sensor.name_1", "not.kat.sensor.name_1", "kat.sensor.name_2",
+            "kat.sensor.name_3", "ket.name.sensor_2"
+        ]
+        expected_candidate_lines = ['kat.sensor.name_1', 'kat.sensor.name_2',
+                                    'kat.sensor.name_3']
+        self.assertIsInstance(sensor_name, str)
+        self.assertIsInstance(sensors_data, list)
```

Comment:
Test for errors, in the case where someone would pass in a different type.
Check:
```python
with self.assertRaises(<expected exception error>) as err:
    ...
```

Read:
- https://stackoverflow.com/questions/6103825/how-to-properly-use-unit-testings-assertraises-with-nonetype-objects
- https://daniel.perez.sh/blog/2017/unittest-assert-raises/